SPECS/edk2/edk2.spec

```diff
@@ -796,6 +799,15 @@ done
 /boot/efi/HvLoader.efi
 
 %changelog
+* Thu May 08 2025 Archana Shettigar <v-shettigara@microsoft.com> - 20240524git3e722403cd16-9
+- Removed CVE-2023-6129 since it applies to powerpc
+
+* Thu Apr 24 2025 Jyoti Kanase <v-jykanase@microsoft.com> - 20240524git3e722403cd16-8
+- Fix CVE-2024-38796
+
+* Wed Apr 23 2025 Archana Choudhary <archana1@microsoft.com> - 20240524git3e722403cd16-7
+- Add patch for CVE-2023-6129, CVE-2024-2511, CVE-2024-4603
```

Comment:
I think we can just drop the latest changelog entry and remove mention of `CVE-2023-6129` completely from the one on 04/23.